NEVER EVER REMOVE THIS PATCH
REBASE IT ON TOP OF THE VERSION YOU'RE UPGRADING

This makes py3-certifi use the system certificates provided by Alpine Linux
instead of the ones provided with py3-certifi instead, this allows us to add
this package as a dependency for other packages without worries.

This is based on the patch used by Debian

diff --git a/certifi/core.py b/certifi/core.py
index 497d938..52868df 100644
--- a/certifi/core.py
+++ b/certifi/core.py
@@ -8,11 +8,12 @@ import os
 import types
 from typing import Union
 
+ALPINE_CA_CERTS_PATH = '/etc/ssl/certs/ca-certificates.crt'
+
 try:
     from importlib.resources import path as get_path, read_text
 
-    _CACERT_CTX = None
-    _CACERT_PATH = None
+    _CACERT_PATH = ALPINE_CA_CERTS_PATH
 
     def where() -> str:
         # This is slightly terrible, but we want to delay extracting the file
@@ -59,10 +60,8 @@ except ImportError:
     # If we don't have importlib.resources, then we will just do the old logic
     # of assuming we're on the filesystem and munge the path directly.
     def where() -> str:
-        f = os.path.dirname(__file__)
-
-        return os.path.join(f, "cacert.pem")
-
+        return ALPINE_CA_CERTS_PATH
 
 def contents() -> str:
-    return read_text("certifi", "cacert.pem", encoding="ascii")
+    with open(where(), "r", encoding="ascii") as data:
+        return data.read()
